changelogs/fragments/5348-fix-vbox-deeply-nested-hostvars.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - virtualbox - Skip parsing values with keys that have both a value and nested data. Skip parsing values that are nested more than two keys deep.
```

Comment:
```suggestion
  - virtualbox - skip parsing values with keys that have both a value and nested data. Skip parsing values that are nested more than two keys deep (https://github.com/ansible-collections/community.general/issues/5332, https://github.com/ansible-collections/community.general/pull/5348).
```

Reply:
```suggestion
  - virtualbox inventory plugin - Skip parsing values with keys that have both a value and nested data. Skip parsing values that are nested more than two keys deep.
```

Reply:
Ah right, thanks for spotting that @russoz! I've updated my suggestion.

Reply:
Change has been added. I've at least tested using the `COLLECTIONS_PATH` config option.